sonar-core/src/main/java/org/sonar/core/platform/SonarQubeVersionProvider.java

```diff
@@ -36,14 +36,18 @@
 
   public SonarQubeVersion provide(System2 system) {
     if (version == null) {
-      try {
-        URL url = system.getResource(FILE_PATH);
-        String versionInFile = Resources.toString(url, StandardCharsets.UTF_8);
-        version = new SonarQubeVersion(Version.parse(versionInFile));
-      } catch (IOException e) {
-        throw new IllegalStateException("Can not load " + FILE_PATH + " from classpath", e);
-      }
+      version = getVersion(system);
     }
     return version;
   }
+
+  public static SonarQubeVersion getVersion(System2 system) {
```

Comment:
I intended to use it in SensorContextTester then realized that sonar-core is not visible from sonar-plugin-api...